src/script/descriptor.cpp

```diff
@@ -816,6 +864,20 @@ class MultisigDescriptor final : public DescriptorImpl
 public:
     MultisigDescriptor(int threshold, std::vector<std::unique_ptr<PubkeyProvider>> providers, bool sorted = false) : DescriptorImpl(std::move(providers), sorted ? "sortedmulti" : "multi"), m_threshold(threshold), m_sorted(sorted) {}
     bool IsSingleType() const final { return true; }
+
+    std::optional<int64_t> ScriptSize() const override {
+        const auto n_keys = m_pubkey_args.size();
+        return 1 + BuildScript(n_keys).size() + BuildScript(m_threshold).size() + 34 * n_keys;
```

Comment:
In c8b0aeb534edce08366176275d5577c1e309eb6f "descriptor: introduce a method to get the satisfaction size"

Same uncompressed pubkey issue as above.

Reply:
This wasn't changed?

Reply:
Oops. Done now.